fast64_internal/oot/scene/exporter/to_c/room_commands.py

```diff
@@ -30,7 +30,8 @@ def getTimeSettingsCmd(outRoom: OOTRoom):
 
 def getWindSettingsCmd(outRoom: OOTRoom):
     return (
-        indent + f"SCENE_CMD_WIND_SETTINGS({', '.join(f'{dir}' for dir in outRoom.windVector)}, {outRoom.windStrength})"
+        indent
+        + f"SCENE_CMD_WIND_SETTINGS({', '.join(f'{dir}' for dir in outRoom.windVector)}, {outRoom.windStrength}),\n"
```

Comment:
:thinking: why only this one and not others
are you sure the issue you saw was specifically about the wind cmd

Reply:
oh it's inconsistent if the `getXxxCmd` functions return with or without `,\n`

Reply:
this one and the actor and object one need ``,\n`` since it's not part of a list that's being used in a join, I can always change that and add the thing after the call but idk it feels weird to me